indico/web/client/styles/base/_palette.scss

```diff
@@ -48,8 +48,8 @@ $yellow: #e99e18;
 $postit-yellow: #ffffa5;
 $light-yellow: #ffe6ba;
 
-$orange: #b14300;
-$bright-orange: #e25300;
+$orange: #f80;
```

Comment:
won't these changes in here affect other parts of indico quite a bit?

Reply:
yeah, "main categories" on the indico front page looks different now. looks a bit strange IMO